css/custom.css

```diff
@@ -6,29 +6,33 @@ body {
 
 .nav-wrapper {
   background-color: #60A550;
+  height: 100px !important;
 }
 
 h1, h2, h3 {
   font-family: "Abril Fatface";
 }
 
-.brand-logo.center {
+.brand-logo.left {
   margin: 0;
-  margin-top: .65%;
-  font-family: 'Abril Fatface';
+  margin-top: .50%;
+  /*font-family: 'Abril Fatface';
   font-size: 2.5em;
   text-transform: capitalize;
-  letter-spacing: 0.1em;
-  height: 50px;
+  letter-spacing: 0.1em;*/
+  height: 80px;
   width: auto;
+  padding-bottom: 7px;
+  padding-left: 15px;
 }
 
 .card-title, a {
   font-family: 'Quicksand';
 }
 
 nav ul a {
-  font-size: 1.2em;
+  font-size: 1.7em !important;
```

Comment:
Do not use `!important`. Codacy is failing because of this. It is not really a good practice.